Buildings/Controls/OBC/CDL/Continuous/PID.mo

```diff
@@ -1,49 +1,69 @@
 within Buildings.Controls.OBC.CDL.Continuous;
 block PID
-  "P, PI, PD, and PID controller with limited output, anti-windup compensation and setpoint weighting"
+  "P, PI, PD, and PID controller"
```

Comment:
I think the controller still have limited output and anti-windup compensation.

Reply:
Correct, but I removed it from the short description as both PID controllers have these features. The short descriptions are

* PID   -  P, PI, PD, and PID controller
* PIDWithReset  -  P, PI, PD, and PID controller with output reset